Makefile

```diff
@@ -100,4 +100,6 @@ CONTROLLER_GEN=$(shell which controller-gen)
 endif
 
 create-installers: set-image
-	kustomize build config/rolebasedcreds > release/rolebased/installer.yaml
+	kustomize build config/installers/rolebasedcreds > release/rolebased/installer.yaml
```

Comment:
This breaks integ tests

Reply:
is it because of RBAC role separation ?

Reply:
We don't use these YAML files to install it during integration tests. It's because I removed RBAC and CRDs from the default base, but didn't add it as a dependency back to the default creds installer.